usb-modem-drivers/pkg.yaml

```diff
@@ -12,6 +12,11 @@ steps:
       - |
         export KERNELRELEASE="{{ .linux_version }}-talos"
 
+        # we need these so tools relying on modules.builtin can find the modules statically compiled into kernel
```

Comment:
would it conflict with same files from `kernel`? would it be same?

Reply:
so depmod won;t generate these files, it has to come from base kernel, since `modules.builtin` contain the list of modules that is statically builtin

Also some tools also check this file to see if module is already loaded

Reply:
moved to the file list, since `install` command handles it better